tests/plugins/pastefromword/generated/onenote.js

```diff
@@ -0,0 +1,38 @@
+/* bender-tags: clipboard,pastefromword */
+/* jshint ignore:start */
+/* bender-ckeditor-plugins: pastefromword,ajax,basicstyles,toolbar */
+/* jshint ignore:end */
+/* bender-include: _lib/q.js,_helpers/promisePasteEvent.js,_helpers/assertWordFilter.js,_helpers/createTestCase.js */
+/* bender-include: _helpers/createTestSuite.js */
+/* global createTestSuite */
+
+( function() {
+	'use strict';
+
+	var config = {
```

Comment:
No need to extract config here, just inline it - [we're doing it very often](https://github.com/ckeditor/ckeditor-dev/blob/b413ecf/tests/plugins/list/list.js#L4-L9) and it's well readable.